### ISSUE TITLE
ansible-lint 25.1.1 unable to find/use git when git URLs are used in requirements.yml

### ISSUE THREAD

**Issue by chriscroome** [ansible/ansible-lint#4498]
##### Summary

With the `25.1.1` version of `ansible-lint` I have this error:

> ansible was NOT installed successfully: could not find/use git

##### Issue Type

- Bug Report

##### OS / ENVIRONMENT

```bash
cat /etc/os-release 
PRETTY_NAME="Debian GNU/Linux 12 (bookworm)"
NAME="Debian GNU/Linux"
VERSION_ID="12"
VERSION="12 (bookworm)"
VERSION_CODENAME=bookworm
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"

ansible-lint --version
ansible-lint 25.1.1 using ansible-core:2.17.7 ansible-compat:25.1.0 ruamel-yaml:0.18.10 ruamel-yaml-clib:0.2.12

which git
/usr/bin/git

git --version
git version 2.39.5
```
- ansible installation method: `pipx` via [this role](https://git.coop/webarch/ansible)
- ansible-lint installation method: `pipx` via [this role](https://git.coop/webarch/ansible)

##### STEPS TO REPRODUCE

Add a git URL to a `requirements.yml` file, for example:

```bash
cat requirements.yml | head -n8
---
# Ansible managed

- name: ansible
  scm: git
  src: https://git.coop/webarch/ansible.git
  version: 3.40.0  # ansible
```

And run `ansible.lint .`

##### Desired Behavior

Previous version of `ansible-lint` worked fine with `requirements.yml` files containing git URLs.

##### Actual Behavior

This behaviour can be reproduced using [this repo](https://git.coop/webarch/localhost).

```bash
ansible-lint .
ERROR    [WARNING]: - ansible was NOT installed successfully: could not find/use git, it
is required to continue with installing https://git.coop/webarch/ansible.git
ERROR! - you can use --ignore-errors to skip failed roles and finish processing the list.
[/]
Got 1 exit code while running: ansible-galaxy role install -r /home/chris/webarch/localhost/requirements.yml --roles-path /home/chris/webarch/localhost/.ansible/roles
```

However if `ansible-galaxy role install` is run then the exit code is `0`:

```bash
ansible-galaxy role install -r /home/chris/webarch/localhost/requirements.yml --roles-path /home/chris/webarch/localhost/.ansible/roles
Starting galaxy role install process
- extracting ansible to /home/chris/ansible
- ansible (3.40.0) was installed successfull
...

echo $?
0
```

There is also no error with a `.pre-commit-config.yaml` containing:

```yaml
---
  - repo: https://github.com/ansible/ansible-lint.git
    rev: v25.1.1
    hooks:
      - id: ansible-lint
        name: Ansible Lint
        language: python
        args:
          - localhost.yml
        additional_dependencies:
          # https://github.com/kellyjonbrazil/jc/releases
          - jc==1.25.3
          # https://github.com/jmespath/jmespath.py/tags
          - jmespath==1.0.1
...
```

```bash
pre-commit run --all
Ansible Lint.............................................................Passed
```

**Comment by chriscroome** [comment]
Thanks @ssbarnea I have tried that but I'm afraid it doesn't seem to make any difference:

```bash
ansible-lint --version
ansible-lint 25.1.1 using ansible-core:2.17.7 ansible-compat:25.1.1 ruamel-yaml:0.18.10 ruamel-yaml-clib:0.2.12

ansible-lint .
ERROR    [WARNING]: - hashicorp was NOT installed successfully: could not find/use git,
it is required to continue with installing
https://git.coop/webarch/hashicorp.git
ERROR! - you can use --ignore-errors to skip failed roles and finish processing the list.
[/]
Got 1 exit code while running: ansible-galaxy role install -r /home/chris/webarch/localhost/requirements.yml --roles-path /home/chris/webarch/localhost/.ansible/roles
```

**Comment by ssbarnea** [comment]
I am unable to reproduce your problem, I did the following:

```shell
> git clone https://git.coop/webarch/localhost.git
> cd localhost
> ansible-lint -vv .
DEBUG    Logging initialized to level 10
INFO     Identified /media/psf/code/a/repro/localhost as project root due .git directory.
DEBUG    Options: Options(_skip_ansible_syntax_check=False, cache_dir=PosixPath('/media/psf/code/a/repro/localhost/.ansible'), colored=True, configured=True, cwd=PosixPath('/media/psf/code/a/repro/localhost'), display_relative_path=True, exclude_paths=['galaxy', 'roles'], format=None, lintables=['.'], list_rules=False, list_tags=False, write_list=[], write_exclude_list=[], parseable=False, quiet=False, rulesdirs=[PosixPath('/media/psf/code/a/ansible-lint/src/ansiblelint/rules')], skip_list=['key-order[task]'], tags=[], verbosity=2, warn_list=['experimental', 'jinja[spacing]', 'fqcn[deep]'], mock_filters=[], mock_modules=[], mock_roles=[], loop_var_prefix=None, only_builtins_allow_collections=[], only_builtins_allow_modules=[], var_naming_pattern=None, offline=False, project_dir='.', extra_vars=None, enable_list=[], skip_action_validation=True, strict=False, rules={}, profile=None, task_name_prefix='{stem} | ', sarif_file=None, config_file='/media/psf/code/a/repro/localhost/.ansible-lint', generate_ignore=False, rulesdir=[], use_default_rules=False, version=False, list_profiles=False, ignore_file=None, max_tasks=100, max_block_depth=20, supported_ansible_also=[])[/]
DEBUG    CWD: /media/psf/code/a/repro/localhost
DEBUG    Logging initialized to level 10
INFO     Collection paths was patch to include extra directories /usr/local/lib/python3.12/dist-packages,/usr/lib/python3/dist-packages,/usr/lib/python3.12/dist-packages,/home/ssbarnea/.ansible/collections,/usr/share/ansible/collections
DEBUG    Loading custom .yamllint config file, this extends our internal yamllint config.
DEBUG    Effective yamllint rules used: {'anchors': {'level': 'error', 'forbid-undeclared-aliases': True, 'forbid-duplicated-anchors': False, 'forbid-unused-anchors': False}, 'braces': {'level': 'error', 'forbid': False, 'min-spaces-inside': 0, 'max-spaces-inside': 1, 'min-spaces-inside-empty': -1, 'max-spaces-inside-empty': -1}, 'brackets': {'level': 'error', 'forbid': False, 'min-spaces-inside': 0, 'max-spaces-inside': 1, 'min-spaces-inside-empty': -1, 'max-spaces-inside-empty': -1}, 'colons': {'level': 'error', 'max-spaces-before': 0, 'max-spaces-after': 1}, 'commas': {'level': 'error', 'max-spaces-before': 0, 'min-spaces-after': 1, 'max-spaces-after': 1}, 'comments': {'level': 'warning', 'require-starting-space': True, 'ignore-shebangs': True, 'min-spaces-from-content': 2}, 'comments-indentation': False, 'document-end': False, 'document-start': {'level': 'warning', 'present': True}, 'empty-lines': {'level': 'error', 'max': 2, 'max-start': 0, 'max-end': 0}, 'empty-values': False, 'float-values': False, 'hyphens': {'level': 'error', 'max-spaces-after': 1}, 'indentation': {'level': 'error', 'spaces': 'consistent', 'indent-sequences': True, 'check-multi-line-strings': False}, 'key-duplicates': {'level': 'error', 'forbid-duplicated-merge-keys': False}, 'key-ordering': False, 'line-length': False, 'new-line-at-end-of-file': {'level': 'error'}, 'new-lines': {'level': 'error', 'type': 'unix'}, 'octal-values': False, 'quoted-strings': False, 'trailing-spaces': {'level': 'error'}, 'truthy': {'level': 'warning', 'allowed-values': ['true', 'false'], 'check-keys': True}}
INFO     Set ANSIBLE_LIBRARY=/media/psf/code/a/repro/localhost/.ansible/modules:/home/ssbarnea/.ansible/plugins/modules:/usr/share/ansible/plugins/modules
INFO     Set ANSIBLE_COLLECTIONS_PATH=/media/psf/code/a/repro/localhost/.ansible/collections:/usr/local/lib/python3.12/dist-packages:/usr/lib/python3/dist-packages:/usr/lib/python3.12/dist-packages:/home/ssbarnea/.ansible/collections:/usr/share/ansible/collections
INFO     Set ANSIBLE_ROLES_PATH=/media/psf/code/a/repro/localhost/.ansible/roles:/media/psf/code/a/repro/localhost/galaxy/roles
INFO     Running ansible-galaxy role install -r /media/psf/code/a/repro/localhost/requirements.yml -vv --roles-path /media/psf/code/a/repro/localhost/.ansible/roles
DEBUG    Starting galaxy role install process
- extracting ansible to /media/psf/code/a/repro/localhost/.ansible/roles/ansible
- ansible (3.40.0) was installed successfully
- extracting apt to /media/psf/code/a/repro/localhost/.ansible/roles/apt
- apt (4.7.0) was installed successfully
- extracting debug to /media/psf/code/a/repro/localhost/.ansible/roles/debug
- debug (1.4.0) was installed successfully
- extracting docker to /media/psf/code/a/repro/localhost/.ansible/roles/docker
- docker (3.7.0) was installed successfully
- extracting edge to /media/psf/code/a/repro/localhost/.ansible/roles/edge
- edge (master) was installed successfully
- extracting firefox to /media/psf/code/a/repro/localhost/.ansible/roles/firefox
- firefox (3.3.1) was installed successfully
- extracting hashicorp to /media/psf/code/a/repro/localhost/.ansible/roles/hashicorp
- hashicorp (main) was installed successfully
- extracting jc to /media/psf/code/a/repro/localhost/.ansible/roles/jc
- jc (2.4.6) was installed successfully
- dependency apt is already installed, skipping.
- extracting jd to /media/psf/code/a/repro/localhost/.ansible/roles/jd
- jd (0.1.0) was installed successfully
- extracting jp to /media/psf/code/a/repro/localhost/.ansible/roles/jp
- jp (3.3.5) was installed successfully
- dependency apt is already installed, skipping.
- extracting kitty to /media/psf/code/a/repro/localhost/.ansible/roles/kitty
- kitty (1.2.0) was installed successfully
- extracting kvm to /media/psf/code/a/repro/localhost/.ansible/roles/kvm
- kvm (main) was installed successfully
- extracting locale to /media/psf/code/a/repro/localhost/.ansible/roles/locale
- locale (2.3.0) was installed successfully
- extracting nodejs to /media/psf/code/a/repro/localhost/.ansible/roles/nodejs
- nodejs (3.2.4) was installed successfully
- dependency apt is already installed, skipping.
- extracting nonfree to /media/psf/code/a/repro/localhost/.ansible/roles/nonfree
- nonfree (main) was installed successfully
- extracting requirements to /media/psf/code/a/repro/localhost/.ansible/roles/requirements
- requirements (1.9.0) was installed successfully
- extracting ssh to /media/psf/code/a/repro/localhost/.ansible/roles/ssh
- ssh (7.3.0) was installed successfully
- extracting systemd to /media/psf/code/a/repro/localhost/.ansible/roles/systemd
- systemd (2.5.2) was installed successfully
- extracting vim to /media/psf/code/a/repro/localhost/.ansible/roles/vim
- vim (1.3.0) was installed successfully
- extracting yarn to /media/psf/code/a/repro/localhost/.ansible/roles/yarn
- yarn (2.2.4) was installed successfully
- dependency apt is already installed, skipping.
- extracting yq to /media/psf/code/a/repro/localhost/.ansible/roles/yq
- yq (1.4.0) was installed successfully

DEBUG    Loading custom .yamllint config file, this extends our internal yamllint config.
DEBUG    Effective yamllint rules used: {'anchors': {'level': 'error', 'forbid-undeclared-aliases': True, 'forbid-duplicated-anchors': False, 'forbid-unused-anchors': False}, 'braces': {'level': 'error', 'forbid': False, 'min-spaces-inside': 0, 'max-spaces-inside': 1, 'min-spaces-inside-empty': -1, 'max-spaces-inside-empty': -1}, 'brackets': {'level': 'error', 'forbid': False, 'min-spaces-inside': 0, 'max-spaces-inside': 1, 'min-spaces-inside-empty': -1, 'max-spaces-inside-empty': -1}, 'colons': {'level': 'error', 'max-spaces-before': 0, 'max-spaces-after': 1}, 'commas': {'level': 'error', 'max-spaces-before': 0, 'min-spaces-after': 1, 'max-spaces-after': 1}, 'comments': {'level': 'warning', 'require-starting-space': True, 'ignore-shebangs': True, 'min-spaces-from-content': 2}, 'comments-indentation': False, 'document-end': False, 'document-start': {'level': 'warning', 'present': True}, 'empty-lines': {'level': 'error', 'max': 2, 'max-start': 0, 'max-end': 0}, 'empty-values': False, 'float-values': False, 'hyphens': {'level': 'error', 'max-spaces-after': 1}, 'indentation': {'level': 'error', 'spaces': 'consistent', 'indent-sequences': True, 'check-multi-line-strings': False}, 'key-duplicates': {'level': 'error', 'forbid-duplicated-merge-keys': False}, 'key-ordering': False, 'line-length': False, 'new-line-at-end-of-file': {'level': 'error'}, 'new-lines': {'level': 'error', 'type': 'unix'}, 'octal-values': False, 'quoted-strings': False, 'trailing-spaces': {'level': 'error'}, 'truthy': {'level': 'warning', 'allowed-values': ['true', 'false'], 'check-keys': True}}
DEBUG    Folder `.` does not look like a role due to missing any of the common subfolders such: tasks, meta, vars, defaults, handlers.
INFO     Loading ignores from .gitignore
INFO     Excluded: .ansible/roles
INFO     Excluded: .git
DEBUG    Passed '.gitlab-ci.yml' positional argument was identified as generic 'yaml' file kind.
DEBUG    Passed '.pre-commit-config.yaml' positional argument was identified as generic 'yaml' file kind.
DEBUG    Passed '.pymarkdown.yml' positional argument was identified as generic 'yaml' file kind.
DEBUG    Passed '.yamllint' positional argument was identified as generic 'yaml' file kind.
DEBUG    data set to None for .ansible/.lock due to being '' (unknown) kind.
DEBUG    data set to None for .gitignore due to being '' (unknown) kind.
DEBUG    Examining .gitlab-ci.yml of type yaml
DEBUG    Running rule internal-error
DEBUG    Running rule load-failure
DEBUG    Running rule parser-error
DEBUG    Running rule warning
DEBUG    Running rule yaml
DEBUG    Running rule args
DEBUG    Running rule avoid-implicit
DEBUG    Running rule command-instead-of-module
DEBUG    Running rule command-instead-of-shell
DEBUG    Running rule complexity
DEBUG    Running rule deprecated-bare-vars
DEBUG    Running rule deprecated-local-action
DEBUG    Running rule deprecated-module
DEBUG    Running rule fqcn
DEBUG    Running rule galaxy
DEBUG    Running rule ignore-errors
DEBUG    Running rule inline-env-var
DEBUG    Running rule jinja
DEBUG    Running rule key-order
DEBUG    Running rule latest
DEBUG    Running rule literal-compare
DEBUG    Running rule loop-var-prefix
DEBUG    Running rule meta-incorrect
DEBUG    Running rule meta-no-tags
DEBUG    Running rule meta-runtime
DEBUG    Running rule meta-video-links
DEBUG    Running rule name
DEBUG    Running rule no-changed-when
DEBUG    Running rule no-free-form
DEBUG    Running rule no-handler
DEBUG    Running rule no-jinja-when
DEBUG    Running rule no-relative-paths
DEBUG    Running rule no-tabs
DEBUG    Running rule package-latest
DEBUG    Running rule partial-become
DEBUG    Running rule playbook-extension
DEBUG    Running rule risky-file-permissions
DEBUG    Running rule risky-octal
DEBUG    Running rule risky-shell-pipe
DEBUG    Running rule role-name
DEBUG    Running rule run-once
DEBUG    Running rule sanity
DEBUG    Running rule schema
DEBUG    Running rule var-naming
DEBUG    Examining .pymarkdown.yml of type yaml
DEBUG    Running rule internal-error
DEBUG    Running rule load-failure
DEBUG    Running rule parser-error
DEBUG    Running rule warning
DEBUG    Running rule yaml
DEBUG    Running rule args
DEBUG    Running rule avoid-implicit
DEBUG    Running rule command-instead-of-module
DEBUG    Running rule command-instead-of-shell
DEBUG    Running rule complexity
DEBUG    Running rule deprecated-bare-vars
DEBUG    Running rule deprecated-local-action
DEBUG    Running rule deprecated-module
DEBUG    Running rule fqcn
DEBUG    Running rule galaxy
DEBUG    Running rule ignore-errors
DEBUG    Running rule inline-env-var
DEBUG    Running rule jinja
DEBUG    Running rule key-order
DEBUG    Running rule latest
DEBUG    Running rule literal-compare
DEBUG    Running rule loop-var-prefix
DEBUG    Running rule meta-incorrect
DEBUG    Running rule meta-no-tags
DEBUG    Running rule meta-runtime
DEBUG    Running rule meta-video-links
DEBUG    Running rule name
DEBUG    Running rule no-changed-when
DEBUG    Running rule no-free-form
DEBUG    Running rule no-handler
DEBUG    Running rule no-jinja-when
DEBUG    Running rule no-relative-paths
DEBUG    Running rule no-tabs
DEBUG    Running rule package-latest
DEBUG    Running rule partial-become
DEBUG    Running rule playbook-extension
DEBUG    Running rule risky-file-permissions
DEBUG    Running rule risky-octal
DEBUG    Running rule risky-shell-pipe
DEBUG    Running rule role-name
DEBUG    Running rule run-once
DEBUG    Running rule sanity
DEBUG    Running rule schema
DEBUG    Running rule var-naming
DEBUG    Examining .pre-commit-config.yaml of type yaml
DEBUG    Running rule internal-error
DEBUG    Running rule load-failure
DEBUG    Running rule parser-error
DEBUG    Running rule warning
DEBUG    Running rule yaml
DEBUG    Running rule args
DEBUG    Running rule avoid-implicit
DEBUG    Running rule command-instead-of-module
DEBUG    Running rule command-instead-of-shell
DEBUG    Running rule complexity
DEBUG    Running rule deprecated-bare-vars
DEBUG    Running rule deprecated-local-action
DEBUG    Running rule deprecated-module
DEBUG    Running rule fqcn
DEBUG    Running rule galaxy
DEBUG    Running rule ignore-errors
DEBUG    Running rule inline-env-var
DEBUG    Running rule jinja
DEBUG    Running rule key-order
DEBUG    Running rule latest
DEBUG    Running rule literal-compare
DEBUG    Running rule loop-var-prefix
DEBUG    Running rule meta-incorrect
DEBUG    Running rule meta-no-tags
DEBUG    Running rule meta-runtime
DEBUG    Running rule meta-video-links
DEBUG    Running rule name
DEBUG    Running rule no-changed-when
DEBUG    Running rule no-free-form
DEBUG    Running rule no-handler
DEBUG    Running rule no-jinja-when
DEBUG    Running rule no-relative-paths
DEBUG    Running rule no-tabs
DEBUG    Running rule package-latest
DEBUG    Running rule partial-become
DEBUG    Running rule playbook-extension
DEBUG    Running rule risky-file-permissions
DEBUG    Running rule risky-octal
DEBUG    Running rule risky-shell-pipe
DEBUG    Running rule role-name
DEBUG    Running rule run-once
DEBUG    Running rule sanity
DEBUG    Running rule schema
DEBUG    Running rule var-naming
DEBUG    Examining .yamllint of type yaml
DEBUG    Running rule internal-error
DEBUG    Running rule load-failure
DEBUG    Running rule parser-error
DEBUG    Running rule warning
DEBUG    Running rule yaml
DEBUG    Running rule args
DEBUG    Running rule avoid-implicit
DEBUG    Running rule command-instead-of-module
DEBUG    Running rule command-instead-of-shell
DEBUG    Running rule complexity
DEBUG    Running rule deprecated-bare-vars
DEBUG    Running rule deprecated-local-action
DEBUG    Running rule deprecated-module
DEBUG    Running rule fqcn
DEBUG    Running rule galaxy
DEBUG    Running rule ignore-errors
DEBUG    Running rule inline-env-var
DEBUG    Running rule jinja
DEBUG    Running rule key-order
DEBUG    Running rule latest
DEBUG    Running rule literal-compare
DEBUG    Running rule loop-var-prefix
DEBUG    Running rule meta-incorrect
DEBUG    Running rule meta-no-tags
DEBUG    Running rule meta-runtime
DEBUG    Running rule meta-video-links
DEBUG    Running rule name
DEBUG    Running rule no-changed-when
DEBUG    Running rule no-free-form
DEBUG    Running rule no-handler
DEBUG    Running rule no-jinja-when
DEBUG    Running rule no-relative-paths
DEBUG    Running rule no-tabs
DEBUG    Running rule package-latest
DEBUG    Running rule partial-become
DEBUG    Running rule playbook-extension
DEBUG    Running rule risky-file-permissions
DEBUG    Running rule risky-octal
DEBUG    Running rule risky-shell-pipe
DEBUG    Running rule role-name
DEBUG    Running rule run-once
DEBUG    Running rule sanity
DEBUG    Running rule schema
DEBUG    Running rule var-naming
DEBUG    Examining .ansible-lint of type ansible-lint-config
DEBUG    Running rule internal-error
DEBUG    Running rule load-failure
DEBUG    Running rule parser-error
DEBUG    Running rule warning
DEBUG    Running rule yaml
DEBUG    Running rule args
DEBUG    Running rule avoid-implicit
DEBUG    Running rule command-instead-of-module
DEBUG    Running rule command-instead-of-shell
DEBUG    Running rule complexity
DEBUG    Running rule deprecated-bare-vars
DEBUG    Running rule deprecated-local-action
DEBUG    Running rule deprecated-module
DEBUG    Running rule fqcn
DEBUG    Running rule galaxy
DEBUG    Running rule ignore-errors
DEBUG    Running rule inline-env-var
DEBUG    Running rule jinja
DEBUG    Running rule key-order
DEBUG    Running rule latest
DEBUG    Running rule literal-compare
DEBUG    Running rule loop-var-prefix
DEBUG    Running rule meta-incorrect
DEBUG    Running rule meta-no-tags
DEBUG    Running rule meta-runtime
DEBUG    Running rule meta-video-links
DEBUG    Running rule name
DEBUG    Running rule no-changed-when
DEBUG    Running rule no-free-form
DEBUG    Running rule no-handler
DEBUG    Running rule no-jinja-when
DEBUG    Running rule no-relative-paths
DEBUG    Running rule no-tabs
DEBUG    Running rule package-latest
DEBUG    Running rule partial-become
DEBUG    Running rule playbook-extension
DEBUG    Running rule risky-file-permissions
DEBUG    Running rule risky-octal
DEBUG    Running rule risky-shell-pipe
DEBUG    Running rule role-name
DEBUG    Running rule run-once
DEBUG    Running rule sanity
DEBUG    Running rule schema
DEBUG    Running rule var-naming
WARNING  Found incompatible custom yamllint configuration (.yamllint), please either remove the file or edit it to comply with:
  - comments.min-spaces-from-content must be 1
  - octal-values.forbid-implicit-octal must be true
  - octal-values.forbid-explicit-octal must be true.

Read https://ansible.readthedocs.io/projects/lint/rules/yaml/ for more details regarding why we have these requirements. Fix mode will not be available.
DEBUG    Attempting to release lock 272836341620496 on /media/psf/code/a/repro/localhost/.ansible/.lock
DEBUG    Lock 272836341620496 released on /media/psf/code/a/repro/localhost/.ansible/.lock

DEBUG    Determined rule-profile order: {'internal-error': (0, 'min'), 'load-failure': (1, 'min'), 'parser-error': (2, 'min'), 'syntax-check': (3, 'min'), 'command-instead-of-module': (4, 'basic'), 'command-instead-of-shell': (5, 'basic'), 'deprecated-bare-vars': (6, 'basic'), 'deprecated-local-action': (7, 'basic'), 'deprecated-module': (8, 'basic'), 'inline-env-var': (9, 'basic'), 'key-order': (10, 'basic'), 'literal-compare': (11, 'basic'), 'jinja': (12, 'basic'), 'no-free-form': (13, 'basic'), 'no-jinja-when': (14, 'basic'), 'no-tabs': (15, 'basic'), 'partial-become': (16, 'basic'), 'playbook-extension': (17, 'basic'), 'role-name': (18, 'basic'), 'schema': (19, 'basic'), 'name': (20, 'basic'), 'var-naming': (21, 'basic'), 'yaml': (22, 'basic'), 'name[template]': (23, 'moderate'), 'name[imperative]': (24, 'moderate'), 'name[casing]': (25, 'moderate'), 'spell-var-name': (26, 'moderate'), 'avoid-implicit': (27, 'safety'), 'latest': (28, 'safety'), 'package-latest': (29, 'safety'), 'risky-file-permissions': (30, 'safety'), 'risky-octal': (31, 'safety'), 'risky-shell-pipe': (32, 'safety'), 'galaxy': (33, 'shared'), 'ignore-errors': (34, 'shared'), 'layout': (35, 'shared'), 'meta-incorrect': (36, 'shared'), 'meta-no-tags': (37, 'shared'), 'meta-video-links': (38, 'shared'), 'meta-version': (39, 'shared'), 'meta-runtime': (40, 'shared'), 'no-changed-when': (41, 'shared'), 'no-changelog': (42, 'shared'), 'no-handler': (43, 'shared'), 'no-relative-paths': (44, 'shared'), 'max-block-depth': (45, 'shared'), 'max-tasks': (46, 'shared'), 'unsafe-loop': (47, 'shared'), 'avoid-dot-notation': (48, 'production'), 'sanity': (49, 'production'), 'fqcn': (50, 'production'), 'import-task-no-when': (51, 'production'), 'meta-no-dependencies': (52, 'production'), 'single-entry-point': (53, 'production'), 'use-loop': (54, 'production')}[/]
DEBUG    Registered VCS backend: bzr
DEBUG    Registered VCS backend: git
DEBUG    Registered VCS backend: hg
DEBUG    Registered VCS backend: svn
DEBUG    Found ansible-lint 25.1.1.dev3 dist
Passed: 0 failure(s), 0 warning(s) on 8 files. Last profile that met the validation criteria was 'production'.
```

**Comment by ssbarnea** [comment]
The `jc was NOT installed successfully: could not find/use git,` might be the cause? I don't see it in my output and I do not have any `jc` command installed.

**Comment by chriscroome** [comment]
I can't now reproduce it either, I guess the `ansible-compat` update did fix it 👍
